redirects/16.0.txt

```diff
@@ -58,13 +58,13 @@ applications/inventory_and_mrp/inventory/management/products/mto.rst application
+applications/inventory_and_mrp/inventory/product_management/product_replenishment/scrap_inventory.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/scrap_inventory.rst                    # /inventory_and_mrp/inventory/product_management/product_replenishment/* -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
 applications/inventory_and_mrp/inventory/management/lots_serial_numbers.rst applications/inventory_and_mrp/inventory/product_management/product_tracking.rst                                                                  # /inventory_and_mrp/inventory/management/lots_serial_numbers -> /inventory_and_mrp/inventory/product_management/product_tracking
 applications/inventory_and_mrp/inventory/management/lots_serial_numbers/differences.rst applications/inventory_and_mrp/inventory/product_management/product_tracking/differences.rst                                          # /inventory_and_mrp/inventory/management/lots_serial_numbers/* -> /inventory_and_mrp/inventory/product_management/product_tracking/*
 applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers.rst applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst                                    # /inventory_and_mrp/inventory/management/lots_serial_numbers/* -> /inventory_and_mrp/inventory/product_management/product_tracking/*
 applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst applications/inventory_and_mrp/inventory/product_management/product_tracking/lots.rst                                                        # /inventory_and_mrp/inventory/management/lots_serial_numbers/* -> /inventory_and_mrp/inventory/product_management/product_tracking/*
 applications/inventory_and_mrp/inventory/management/lots_serial_numbers/expiration_dates.rst applications/inventory_and_mrp/inventory/product_management/product_tracking/expiration_dates.rst                                # /inventory_and_mrp/inventory/management/lots_serial_numbers/* -> /inventory_and_mrp/inventory/product_management/product_tracking/*
 applications/inventory_and_mrp/inventory/management/inventory_adjustments.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst                                             # /inventory_and_mrp/inventory/management/inventory_adjustments -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products
-applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst                              # /inventory_and_mrp/inventory/management/inventory_adjustments/count_products -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
```

Comment:
I'm not sure why you removed the existing redirect for `count_products.rst `, please revert this so if any links to the _old_ file location exist, the user is properly redirected :)
```
applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst                              # /inventory_and_mrp/inventory/management/inventory_adjustments/count_products -> /inventory_and_mrp/inventory/warehouses_storage/inventory_management/*
```

Reply:
Thank you SO much for catching this, Sam! I got confused by the long paths when moving things around. Definitely did NOT intend to remove this line. Thank you SOSOSO much, again!